website/docs/r/mssql_elasticpool.html.markdown

```diff
@@ -67,6 +67,8 @@ The following arguments are supported:
 
 * `max_size_gb` - (Optional) The max data size of the elastic pool in gigabytes. Conflicts with `max_size_bytes`.
 
+~> **Note:** `max_size_gb` is required when `sku` is specified.
```

Comment:
the `sku` field is required - however one of either `max_size_gb` or `max_size_bytes` must be configured - so can we remove this:

```suggestion
```

Reply:
updated